File: 10_Days_of_Statistics/Central_limit_theorem_III.cpp
#include <iostream>
#include <math.h>
#include <iomanip>
#include <vector>


using namespace std;

int main() {
    int n;
    cin >> n;
    float mu, std, p, z;
    cin >> mu;
    cin >> std;
    cin >> p;
    cin >> z;
    cout<<std::fixed<< std::setprecision(2) << mu - z*std/sqrt(n) <<endl;
    cout<<std::fixed<< std::setprecision(2) << mu + z*std/sqrt(n) <<endl;
   return 0;
}

File: 10_Days_of_Statistics/Central_limit_theorem_III.py
import math

def CLT(z , mu, sigma, n):
    return mu + z*sigma/math.sqrt(n)


n = int(input())
mu = int(input())
std = int(input())
p = float(input())
z = float(input())

print(round((CLT(-z, mu, std, n)), 2))
print(round((CLT(z, mu, std, n)), 2))
